gprofiler/python.py

```diff
@@ -280,31 +276,67 @@ def stop(self):
             logger.info("Finished profiling Python processes with PyPerf")
 
 
-_profiler_class = None
+class PythonProfiler(ProfilerInterface):
+    """
+    Controls PySpyProfiler & PythonEbpfProfiler as needed, providing a clean interface
+    to GProfiler.
+    """
 
+    def __init__(
+        self,
+        frequency: int,
+        duration: int,
+        stop_event: Event,
+        storage_dir: str,
+        python_mode: str,
+    ):
+        # not calling my c'tor - we're just a proxy class
```

Comment:
Is it relevant? I use `ProfilerInterface` here, not `ProfilerBase`

Reply:
Doesn't really matter but if someone adds a constructor later..

Reply:
Will change to the interface

Reply:
Hmm I already use the interface. So comment is irrelevant.